doc/man3/BN_set_bit.pod

```diff
@@ -33,8 +33,10 @@ error occurs if B<a> is shorter than B<n> bits.
 BN_is_bit_set() tests if bit B<n> in B<a> is set.
 
 BN_mask_bits() truncates B<a> to an B<n> bit number
-(C<a&=~((~0)E<lt>E<lt>n)>).  An error occurs if B<a> already is
-shorter than B<n> bits.
+(C<a&=~((~0)E<lt>E<lt>n)>). An error occurs if B<a> is already shorter than
+B<n> bits. Note that this check depends on the platform's word size, and the
```

Comment:
Error also occurs if n is less than 0.  Given that, does it make sense to indicate the return value can be safely ignored?

Reply:
@nhorman That's indeed a good point I didn't consider. I have updated the section accordingly:

```
BN_mask_bits() truncates B<a> to an B<n> bit number
(C<a&=~((~0)E<lt>E<lt>n)>). An error occurs if B<a> is already shorter than
B<n> bits or if B<n> is negative. Note that this bit check depends on the
platform's word size. Use L<BN_num_bits(3)> to determine the exact number of
bits if needed.
```